web/src/components/organisms/Project/Schema/hooks.ts

```diff
@@ -27,8 +26,8 @@ export default () => {
   const [collapsed, collapse] = useState(false);
 
   useEffect(() => {
-    if (!modelId && currentModel) {
-      navigate(`/workspace/${workspaceId}/project/${projectId}/schema/${currentModel.id}`);
+    if (!modelId) {
+      setCurrentModel();
```

Comment:
Since you are trying to keep the model setting and updating logic in the ModelsMenu maybe you don't need to be checking for the modelId here since this should be handled by the ModelsMenu, right?
Also, do you think we should be using `useParams` for any of the IDs outside of the Header (for workspace and project IDs) and ModelsMenu (for model ID)??

Reply:
@KaWaite , done.

I think so, so we can keep one place for updating their stated, and also one instance selected as a time, also it will easier for developing because developer can use the state directly without thinking much about the way it is fetched.